Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml

```diff
@@ -33,10 +29,9 @@
                             <displayCond>FIELD:parentRec.CType:=:servicebw2_organizationalunitsshow</displayCond>
                             <config>
                                 <type>group</type>
-                                <internal_type>db</internal_type>
                                 <allowed>pages</allowed>
                                 <size>1</size>
-                            </config>
+                            </config>y
```

Comment:
y

Reply:
this is rubbish!! and embarassing because of keyboard errors.